pkg/api/cluster_handler.go

```diff
@@ -66,16 +71,17 @@ func (e *etcdHandler) GetObjectStoreConnectionInfo(name string) (*model.ObjectSt
 		return nil, true, err
 	}
 
-	host, ipEndpoint, found, err := rgw.GetRGWEndpoints(e.context.EtcdClient, clusterInventory)
+	host, ipaddress, found, err := rgw.GetRGWEndpoints(e.context.EtcdClient, clusterInventory)
 	if err != nil {
 		return nil, !util.IsEtcdKeyNotFound(err), err
 	} else if !found {
 		return nil, false, fmt.Errorf("failed to find rgw endpoints")
 	}
 
 	s3Info := &model.ObjectStoreConnectInfo{
-		Host:       host,
-		IPEndpoint: ipEndpoint,
+		Host:      host,
+		IPAddress: ipaddress,
+		Ports:     []int32{53390},
```

Comment:
is this a default port that ceph assigned durign creation? If so, i would add it as a const.

Reply:
this is going away with standalone so i didn't worry about it